setup.py

```diff
@@ -24,6 +24,7 @@ def readme():
     'google-auth>=1.0.0',
     'google-auth-httplib2>=0.0.1',
     'google-auth-oauthlib>=0.0.1',
+    'google-cloud-bigquery==0.26.0',
```

Comment:
>=

Reply:
I believe the code as-written would also function on 0.27.0, so a range like `google-cloud-bigquery>=0.26.0,<0.28.0` would be better.

Note that [0.28.0 is coming soon](https://github.com/GoogleCloudPlatform/google-cloud-python/pull/4258) which includes many breaking changes, but also fixes the timeout issue mentioned above.

Reply:
Can you clarify? Change `==` to `=` causes setup to fail due to parse error.

Reply:
@tswast 👍 Switched to your suggestion